cloud/blockstore/tests/csi_driver/test.py

```diff
@@ -419,6 +458,43 @@ def test_node_volume_expand():
             env.csi.unpublish_volume(pod_id, volume_name)
         except subprocess.CalledProcessError as e:
             log_called_process_error(e)
+        try:
+            env.csi.unstage_volume(volume_name)
+        except subprocess.CalledProcessError as e:
+            log_called_process_error(e)
+        try:
+            env.csi.delete_volume(volume_name)
+        except subprocess.CalledProcessError as e:
+            log_called_process_error(e)
+        cleanup_after_test(env)
+
+
+def test_publish_volume_twice_on_the_same_node():
+    env, run = init(vm_mode=True)
+    try:
+        volume_name = "example-disk"
+        volume_size = 1024 ** 3
+        pod_name1 = "example-pod-1"
+        pod_name2 = "example-pod-2"
+        pod_id1 = "deadbeef1"
+        pod_id2 = "deadbeef2"
+        env.csi.create_volume(name=volume_name, size=volume_size)
+        env.csi.stage_volume(volume_name)
+        env.csi.publish_volume(pod_id1, volume_name, pod_name1)
+        env.csi.publish_volume(pod_id2, volume_name, pod_name2)
+    except subprocess.CalledProcessError as e:
+        log_called_process_error(e)
+        raise
+    finally:
+        try:
```

Comment:
Let us make a simple context manager for that:
```
@contextlib.contextmanager
def called_process_error_logged():
    try:
         yield
    except subprocess.CalledProcessError as e:
           log_called_process_error(e)
```

Reply:
Done